src/android/AndroidSaveDataHelper.cpp

```diff
@@ -0,0 +1,76 @@
+
+//#if defined(__ANDROID__)
```

Comment:
Should only be android? 
Can we also put perhaps a platform link to documentation, incase it gets updated in the future?